.github/docker/Dockerfile-manylinux.template

```diff
@@ -26,26 +26,36 @@ FROM quay.io/pypa/manylinux{{ TYPE }}_{{ ARCH }}:latest
 
 ARG PY_MINORS="7 8 9 10"
 
+COPY requirements.txt /tmp/requirements.txt
+
+{{ EXTRA_PRE }}
+
 # Enable rpmfusion for additional packages
 RUN yum update -y && \
-  yum localinstall -y \
+  yum localinstall -y --skip-broken \
     https://mirrors.rpmfusion.org/free/el/rpmfusion-free-release-$(rpm --eval %{centos_ver}).noarch.rpm \
     https://mirrors.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-$(rpm --eval %{centos_ver}).noarch.rpm && \
   yum install -y \
     redland-devel \
     rasqal-devel \
-    hdf5-devel \
```

Comment:
Why do you remove hdf5-devel? Are there some issues with the package? 
If not, it would be nice to have it around such that we can enable the hdf5 storage plugin

Reply:
It's only removed for `manylinux2014_i686` (32-bit) due to no hdf5 package existing for this distribution.